Makefile

```diff
@@ -512,6 +514,27 @@ verify-gen: generate ## Verfiy go generated files are up to date
 verify-conversions: $(CONVERSION_VERIFIER) ## Verifies expected API conversion are in place
 	$(CONVERSION_VERIFIER)
 
+.PHONY: verify-container-images
+verify-container-images: ## Verify container images
+	TRACE=$(TRACE) ./hack/verify-container-images.sh $(TRIVY_VER)
+
+.PHONY: verify-govulncheck
+verify-govulncheck: $(GOVULNCHECK) ## Verify code for vulnerabilities
+	$(GOVULNCHECK) ./... && R1=$$? || R1=$$?; \
+	$(GOVULNCHECK) -C "$(TOOLS_DIR)" ./... && R2=$$? || R2=$$?; \
+	if [ "$$R1" -ne "0" ] || [ "$$R2" -ne "0" ]; then \
+		exit 1; \
+	fi
+
+.PHONY: verify-security
+verify-security: ## Verify code and images for vulnerabilities
```

Comment:
Can we add this target into the `verify` to run in the security. But my only worry is that we keep checking for every PR which may end up failing even though changes doesn't contain the issues because the vulnerabilities can arise anytime based on the external dependencies.

Reply:
I see they are calling weekly via GH action https://github.com/kubernetes-sigs/cluster-api/pull/9144

Reply:
Yes will add that.

Reply:
```
prajyotparab@Prajyots-MacBook-Pro tools % make bin/trivy
go build -tags=capibmtools -o bin/trivy github.com/aquasecurity/trivy/cmd/trivy
no required module provides package github.com/aquasecurity/trivy/cmd/trivy; to add it:
        go get github.com/aquasecurity/trivy/cmd/trivy
make: *** [bin/trivy] Error 1
prajyotparab@Prajyots-MacBook-Pro tools % go get github.com/aquasecurity/trivy/cmd/trivy
...
...
...
prajyotparab@Prajyots-MacBook-Pro tools % make bin/trivy
go build -tags=capibmtools -o bin/trivy github.com/aquasecurity/trivy/cmd/trivy
# oras.land/oras-go/pkg/auth/docker
../../../../go/pkg/mod/oras.land/oras-go@v1.2.3/pkg/auth/docker/login.go:86:39: cannot use remote (variable of type *"github.com/docker/docker/registry".Service) as "github.com/docker/docker/registry".Service value in argument to c.loginWithTLS
make: *** [bin/trivy] Error 1
```

Reply:
fixed.